assets/sass/layouts/_responsive.scss

```diff
@@ -0,0 +1,27 @@
+@media (max-width: 1130px) {
```

Comment:
please create a variable for this media query in the _grid-settings.scss file here:
https://github.com/moxie-leean/Leean/blob/master/assets/sass/_grid-settings.scss

and use @include media($your-variable-name){
/* code here */
}

here is an example: https://github.com/moxienyc/Printmornyc.com/blob/develop/wp-content/themes/Printmornyc/assets/sass/modules/_my-address.scss#L24